elements/kubelet/install.d/70-kubelet

```diff
@@ -29,7 +29,6 @@ kernel.panic = 10
 kernel.panic_on_oops = 1
 net.bridge.bridge-nf-call-ip6tables = 1
 net.bridge.bridge-nf-call-iptables = 1
-net.ipv4.conf.all.rp_filter = 0
```

Comment:
The default for this is `2` I believe, but I think both Calico and Cilium would like this to be `0`.  Any reason for the change?